MoesifNet7Example/MoesifNet7Example/appsettings.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "MoesifOptions": {
+    "ApplicationId": "eyJhcHAiOiI1MTk6MzQ1IiwidmVyIjoiMi4xIiwib3JnIjoiNDIwOjMyMCIsImlhdCI6MTY5NjExODQwMH0.Ved_dF4sHK-E2kTCHhK3pj4VNk6krgJC-w8gxvwxEQE",
```

Comment:
@qmjava we can not check-in any key/id/secrets.

Reply:
yeah, forgot to save the change